src/modules/integrator-guide/pages/authentication/authenticate-external/with-corteza.adoc

```diff
@@ -5,8 +5,7 @@ include::ROOT:partial$variables.adoc[]
 :description: Learn how to authenticate {PRODUCT_NAME} instances using different {PRODUCT_NAME} instances
 
 [#provider]
-== Setup {PRODUCT_NAME} you will use to authenticate with
-//Editor's note: shorten the title and make it sound prettier. Use the infinitive form of verbs or better yet, avoid using verbs in titles.
```

Comment:
What do you think of this one?

Reply:
"Configuring the Authentication Provider"? I think it's a good suggestion. Go with this instead.